tests/ui_tools/test_boxes.py

```diff
@@ -337,6 +341,7 @@ def test__set_stream_write_box_style_markers(self, write_box, stream_id,
         write_box._set_stream_write_box_style(write_box, stream_name)
 
         assert write_box.header_write_box[0].text == expected_marker
+        assert expected_color in str(write_box.header_write_box[0].get_text())
```

Comment:
Yep. Actually the `.get_text()` attribute gives a tuple object `('#', [('color', 1)])`, so I thought it would be clean to stringify it and search for the expected color.